src/Game.tsx

```diff
@@ -214,8 +195,13 @@ export function Game({ pokemonOne, pokemonTwo }: Props): JSX.Element {
           ? pokemonOne
           : pokemonTwo
       );
-      initialBallState.ballStepX = getArrayRandomElement([-2, 2]);
-      initialBallState.ballStepY = getArrayRandomElement([-1, 1]);
+      setBallState((prevState) => {
+        return {
+          ...prevState,
```

Comment:
remove redundant `setBallState` in begin of `useEffect`

Replace update function by merging `initialState` and new values